README.md

```diff
@@ -55,19 +201,24 @@ You have the possibility to customize `GenerateIntl` annotation with the followi
 - defaultFlavor: default flavor of your project, default to 'default'
 - locales: List of supported locales of your project
 - flavors: List of flavors of your project
-- generateFlutterDelegate: If your using pure dart project, you'll need to disable flutter related code, default to true
 - codegenMode: mode to pass to underlaying intl_generator, default to 'release'
 - useDeferredLoading: either to use deferred loading for localization files, default to true
 - arbSuppressMetaData: suppress meta data when generating the ARB file, default to false
 - arbSuppressLastModified: suppress last modified when generating the ARB file, default to false
 - arbIncludeSourceText: whether to include source_text in messages, default to false
 
-## Localization from ARB
+## Limitations
+
+As this package is based on intl, it have some of his limitations, the main one is that intl use a global internal map to find translations in the correct locale.
```

Comment:
As this package is based on intl, it inherits some of his limitations, the main one is that intl's use a global internal map to find translations in the correct locale.